<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>519</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>477</width>
    <height>411</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
	
	background-color: rgb(255, 255, 255);
	border : 0px solid #ffffff;
}</string>
  </property>
  <widget class="QTextEdit" name="login_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="logs_">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="delay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>428</x>
     <y>137</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="db_username">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>94</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="db_passwrd">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>133</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="db_host">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>55</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="db_port">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>94</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="db_name">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>201</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>81</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>delay</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>48</y>
     <width>81</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>53</y>
     <width>81</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Db name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>87</y>
     <width>81</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>126</y>
     <width>81</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>87</y>
     <width>81</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tst_con">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Test Con</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="quiryline">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>458</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exic">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>173</y>
     <width>87</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>exicute</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
